src/DacpacTool/PackageBuilder.cs

```diff
@@ -73,11 +73,22 @@ public void AddInputFile(FileInfo inputFile)
             if (inputFile.Directory.Name.Equals("rules", StringComparison.OrdinalIgnoreCase)
                 && inputFile.Extension.EndsWith(".dll", StringComparison.OrdinalIgnoreCase))
             {
-                return;
+                return true;
             }
 
             Console.WriteLine($"Adding {inputFile.FullName} to the model");
-            Model.AddOrUpdateObjects(File.ReadAllText(inputFile.FullName), inputFile.FullName, new TSqlObjectOptions());
+
+            try
+            {
+                TSqlObjectOptions sqlObjectOptions = new TSqlObjectOptions();
+                Model.AddOrUpdateObjects(File.ReadAllText(inputFile.FullName), inputFile.FullName, new TSqlObjectOptions());
+                return true;
+            }
+            catch (DacModelException dex)
+            {
+                Console.WriteLine(dex.Format(inputFile.Name));
```

Comment:
This might be beyond the scope of this PR, but I'm now wondering why this isn't using the `IConsole` abstraction instead of directly writing to the `Console`.